machine_management/user_infra/adding-compute-user-infra-general.adoc

```diff
@@ -0,0 +1,32 @@
+[id="adding-compute-user-infra-general"]
+= Adding compute machines to clusters with user-provisioned infrastructure
+include::modules/common-attributes.adoc[]
+:context: adding-compute-user-infra-general
+
+toc::[]
+
+You can add compute machines to a cluster on user-provisioned infrastructure either as part of the installation process or after installation. The post-installation process requires some of the same configuration files and parameters that were used during installation.
+
+== Adding compute machines to Amazon Web Services
+
+To add more compute machines to your {product-title} cluster on Amazon Web Services (AWS), see xref:../../machine_management/user_infra/adding-aws-compute-user-infra.adoc#adding-aws-compute-user-infra[Adding compute machines to AWS by using CloudFormation templates].
+
+== Adding compute machines to Microsoft Azure
+
+To add more compute machines to your {product-title} cluster on Microsoft Azure, see xref:../../installing/installing_azure/installing-azure-user-infra.adoc#installation-creating-azure-worker_installing-azure-user-infra[Creating additional worker machines in Azure] in the _Installing a cluster on Azure using ARM templates_ documentation.
```

Comment:
Not critical, but when I saw these "in the _page title_" after the link, I was expecting it to be an external link. Just because you're linking directly to a module, I don't think it's necessary to specify this.